Ansible/playbook.yml

```diff
@@ -0,0 +1,79 @@
+- name: Run todo app on VM
+  hosts: webservers
+  remote_user: ec2-user
+  vars_prompt: 
+  - name: api_key
+    prompt: Enter the Trello API Key...
+    private: no
+  
+  - name: trello_token
+    prompt: Enter the Trello Token... 
+    private: no 
+
+  - name: trello_id 
+    prompt: Enter the Trello Board Id... 
+    private: no
+
+  tasks: 
+  - name: Install Git 
+    ansible.builtin.yum:
+      name: git
+      state: latest
+    become: yes
+  
+  - name: Install Python 3
+    ansible.builtin.yum:
+      name: python3
+      state: latest
+    become: yes
+  
+  - name: Install Poetry
+    ansible.builtin.shell:
+      cmd: curl -sSl https://install.python-poetry.org | python3 -
+  #create method could be added here
```

Comment:
e.g. `creates: /home/ec2-user/.local/bin/poetry`

This lets Ansible know it can skip the task if that file already exists.